### DISCUSSION TITLE
Neovim weird mouse selection behaviour

### DISCUSSION THREAD

**Discussion by davidnoronha1** [wezterm/wezterm#823] [Q&A] [answered]
When I use the mouse to select text, vim puts me in visual mode but the odd thing is in neovim and wezterm
it will only select chunks of text entire words when using the mouse despite being in the normal character wise visual mode. Vim does not do this.

Ive compared it to st and ive only found this happening when using neovim and wezterm (I tested without any user config)

To enable mouse support I set
```
set mouse=a
```
and in vim (neovim removed this setting)
```
set ttymouse=sgr
```

Im not sure where the issue lies probably with neovim but.. im not sure how to proceed.

**Comment by wez** [comment, marked as answer]
It is technically a neovim bug: it seems as though it interprets drag events with the same coordinates as a double click.
I've made wezterm avoid generating drag events with the same coordinates in 2e924c9627fba2178155bdf2177eda84e1daf99a
which should show up in the nightly builds in ~30 minutes from now.

**Reply by davidnoronha1** [reply]
I'll test it then, thank you :D
